ocaml/doc/xenserver/branding.js

```diff
@@ -63,6 +63,8 @@ function get_release_name(s)
 		return 'XenServer 6.5 SP1';
 	case 'dundee':
 		return 'XenServer 7.0';
+	case 'ely':
```

Comment:
The branch `1.14-ppx` is not part of `ely/lcm` and goes only into the Honolulu release. Is this still correct?